lib/interactive.dart

```diff
@@ -39,12 +41,18 @@ class Interactive {
     _render();
   }
 
-  void _updateFilterOption(String value) {
+  void _updateFilterOption(String value) async {
     value = value == "null" ? null : value;
     _selected.updateOption(value);
+    await _loadPeople();
     _render();
   }
 
+  void _loadPeople() async {
+    var people = await fb.readPeople(_selected.filter, _selected.option);
+    print("Loaded ${people.length} people");
```

Comment:
Could we factor this into a log statement pls

Reply:
Sure, will add it in the next PR